js/beerxml-to-beerjson.js

```diff
@@ -185,105 +179,210 @@ const hop_bill = r =>
   }))
 
 const fermentable_bill = r =>
-  _.map(
-    getArrayNode(_.get(r, ['FERMENTABLES', 'FERMENTABLE'])),
-    fermentable => ({
-      name: fermentable['NAME'],
-      type: _.lowerCase(fermentable['TYPE']),
-      color: {
-        unit: 'Lovi',
-        value: Number(fermentable['COLOR'])
+  map(getArrayNode(get(r, ['FERMENTABLES', 'FERMENTABLE'])), fermentable => ({
+    name: fermentable['NAME'],
+    type: getFermentableType(lowerCase(fermentable['TYPE'])),
+    color: {
+      unit: 'Lovi',
+      value: Number(fermentable['COLOR'])
+    },
+    amount: {
+      unit: 'kg',
+      value: Number(fermentable['AMOUNT'])
+    },
+    origin: fermentable['ORIGIN'],
+    supplier: fermentable['SUPPLIER'],
+    group: 'base',
+    yield: {
+      potential: {
+        unit: 'sg',
+        value: potential(Number(fermentable['YIELD']))
+      }
+    },
+    ...(!isEmpty(fermentable['ADD_AFTER_BOIL'])
+      ? {
+          add_after_boil: parseBool(fermentable['ADD_AFTER_BOIL'])
+        }
+      : {})
+  }))
+
+const DEFAULT_ATTENUATION = 75
+
+const importCultureAdditions = recipe => {
+  const cultures = get(recipe, 'YEASTS.YEAST')
+
+  if (isEmpty(cultures)) {
+    return []
+  }
+
+  return map(getArrayNode(cultures), culture => ({
+    name: culture.NAME,
+    type: toLower(culture.TYPE),
+    form: toLower(culture.FORM),
+    amount: {
+      unit: parseBool(culture.AMOUNT_IS_WEIGHT) ? 'kg' : 'l',
+      value: Number(culture.AMOUNT)
+    },
+    attenuation: {
+      unit: '%',
+      value: isEmpty(culture.ATTENUATION)
+        ? DEFAULT_ATTENUATION
+        : Number(culture.ATTENUATION)
+    }
+  }))
+}
+
+const checkRequiredFields = (obj, fieldList) =>
+  every(fieldList, field => !isEmpty(obj[field]))
+
+const importStyle = recipe => {
+  const { STYLE } = recipe
+
+  if (
+    isEmpty(STYLE) ||
+    !checkRequiredFields(STYLE, [
```

Comment:
Why not drop STYLE to the next line?

Reply:
Source code is formatted with the prettier plugin before commit automatically. We could change the prettier plugin settings, but it is a matter of preference.